line/line.mjs

```diff
@@ -1,27 +1,100 @@
-import * as utils from '../utilities/shared.mjs'
+import * as utils from '../utilities/shared.mjs';
 
-export async function findLineById(id = null) {
-  let line = null
-
-  if (!utils.validateID(id)) {
-    return line
+export async function findLineById(id = null, options = {}) {
+  if (id === null || id === undefined || !utils.validateID(id)) {
+    return { statusCode: 400, headers: { 'Content-Type': 'application/json' }, body: null };
   }
 
   const mockPause = new Promise((resolve) => {
     setTimeout(() => {
-      resolve(null)
-    }, 1500)
-  })
+      resolve(null);
+    }, 1500);
+  });
 
   const linesArray = [
-    { id: 123, text: "Hey TPEN Works on 123" },
-  ]
+    {
+      id: 123,
+      text: 'Hey TPEN Works on 123',
+      '@context': 'http://t-pen.org/3/context.json',
+      '@type': 'Annotation',
+      creator: 'https://store.rerum.io/v1/id/hash',
+      project: '#ProjectId',
+      canvas: 'https://example.com/canvas.json',
+      layer: '#AnnotationCollectionId',
+      viewer: 'https://static.t-pen.org/#ProjectId/#PageId/#LineId123',
+      license: 'CC-BY',
+    },
+  ];
+
+  let line = linesArray.find((line) => line.id === id) || (await mockPause);
+
+  if (line === null || line.id !== id) {
+    return { statusCode: 404, body: `TPEN 3 line "${id}" does not exist.` };
+  }
+
+  switch (options.text) {
```

Comment:
A single conditional doesn't need a switch, but this part is all fluid anyway.

Reply:
Updated with the if in the code for the blob